projects/js-toolkit/packages/js-api/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+	"extends": "../../tsconfig.json",
+	"compilerOptions": {
+		"composite": true,
+		"lib": ["DOM", "ES2020"],
+		"outDir": ".",
+		"rootDir": ".",
```

Comment:
should this be `./src` to follow our other patterns?

Reply:
No, it can't be because of how typescript works when this package is consumed :-(. The problem is that in typescript you can either:

1. Designate a single global .d.ts file as your package entry point, or 
2. Let it work by default (but then everything needs to be at the root of the package)

And the problem with 1 is that nobody knows how such file can be generated :shrug: . The official docs make a reference to a "bundled .d.ts file", but there's no explanation anywhere on how to build such file.

It could be that I haven't found it, but I can tell you I tried hard :-/

Reply:
We could also create src and lib directories, then create a prepublish script to undo that, but it sounds risky bc we can break releases. Though I have to admit I don't like having everything hanging around when doing local development because even with .gitignore and .eslintignore properly configured, it's a bit of a hotchpotch...

Reply:
Found this https://github.com/TypeStrong/dts-bundle, but it's experimental...